src/core/reference/include/ngraph/runtime/reference/topk.hpp

```diff
@@ -30,12 +30,11 @@ inline bool compare_max(const std::tuple<T, U>& a, const std::tuple<T, U>& b) {
 #if defined(__GNUC__)
```

Comment:
```
    if (std::get<0>(a) == std::get<0>(b)) {
        return std::get<1>(a) < std::get<1>(b);
    }
```
If I understand it right, the above code block assures sorting to be stable regardless of the attribute of "stable". I think the behavior is right. But since now "stable" becomes one of the attributes, and we don't need to pass the attributes "stable" because we assure stable anyway. Maybe we should add some description, so that people know the attribue of "stable" has been applied without any confusion.